arm-software/linux/docs/OpenMPSupport.md

```diff
@@ -0,0 +1,87 @@
+<!--===- docs/OpenMPSupport.md
+
+   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+   See https://llvm.org/LICENSE.txt for license information.
+   SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+-->
+
+# Clang OpenMP support
+
+Refer to the (status page](https://releases.llvm.org/20.1.0/tools/clang/docs/OpenMPSupport.html) for details on Clang OpenMP support.
+
+# Flang OpenMP Support
+
+```{contents}
```

Comment:
I'm not sure what's the plan for those docs. Other files in this directory don't have this `local:` thing, what is the purpose of this markings?

Reply:
This generates a table of contents in HTML. Will remove for now.